src/corelibs/U2View/src/ov_sequence/view_rendering/SequenceViewAnnotatedRenderer.cpp

```diff
@@ -231,6 +275,52 @@ void SequenceViewAnnotatedRenderer::drawBoundedText(QPainter& p, const QRect& r,
     p.drawText(r, Qt::TextSingleLine | Qt::AlignCenter, text.left(prefixLen));
 }
 
+int SequenceViewAnnotatedRenderer::getCutPosition(const U2Region& r, const QString& cutQual, bool isAnnotationDirect, bool isCutDirect) const {
+    bool hasDirect = false;
+    bool hasComplement = false;
+    int cutDirect = 0;
+    int cutComplement = 0;
+    if (!cutQual.isEmpty()) {
+        int complSplit = cutQual.indexOf('/');
+        if (-1 != complSplit) {
+            cutDirect = cutQual.left(complSplit).toInt(&hasDirect);
+            cutComplement = cutQual.mid(qMin(cutQual.length(), complSplit + 1)).toInt(&hasComplement);
```

Comment:
Why do you need this qMin()? `complSplit` is always within the `cutQual` right?

Reply:
I do not know, maybe some legacy, removed